clang/test/CIR/CodeGen/X86/compressStore_builtin.c

```diff
@@ -0,0 +1,5 @@
+#include <immintrin.h>
```

Comment:
This test should be moved to `clang/test/CIR/CodeGenBuiltins/X86/avx512vl-builtins.c` which is also being added in https://github.com/llvm/llvm-project/pull/169582. The test will need RUN lines and CIR, LLVM, and OGCG checks added.